plugins/connection/incus.py

```diff
@@ -9,10 +9,13 @@
 DOCUMENTATION = r"""
 author: Stéphane Graber (@stgraber)
 name: incus
-short_description: Run tasks in Incus instances using the Incus CLI
+short_description: Run tasks in Incus instances and Windows VMs using the Incus CLI
 description:
-  - Run commands or put/fetch files to an existing Incus instance using Incus CLI.
+  - Run commands or put/fetch files to an existing Incus instances and Windows VMs using Incus CLI.
 version_added: "8.2.0"
+notes:
+  - When using this collection for Windows virtual machines, add C(ansible_shell_type) to C(powershell) or C(cmd) as a variable to the host in
+    the inventory.
```

Comment:
Maybe `set C(ansible_shell_type) to ...` instead of `add C(ansible_shell_type) to ...`?